resources/lang/en/default.php

```diff
@@ -3,6 +3,7 @@
 return [
     'heading' => 'Lock Screen',
     'user_menu_title' => 'Lock Screen',
+    'password' => 'Password',
```

Comment:
Can you change the Key name to "password_field" to able to identify that it is use for the the field.

Reply:
Done.. now `password` is under `fields`